dnf/module/module_base.py

```diff
@@ -33,6 +33,11 @@
 MODULE_TABLE_HINT = _("\n\nHint: [d]efault, [e]nabled, [x]disabled, [i]nstalled")
 MODULE_INFO_TABLE_HINT = _("\n\nHint: [d]efault, [e]nabled, [x]disabled, [i]nstalled, [a]ctive")
 
+
+def _profile_sorter(profile):
```

Comment:
How about naming the function ``_profile_comparison_key`` (or something like this), as it is not the sorter (does not sort anything).